src/components/table/demo/filtered/style.css

```diff
@@ -1,3 +1,6 @@
 :host {
 	display: block;
 }
+smoothly-table-cell:nth-of-type(2){
```

Comment:
Is'nt this quite arbitrary?. in tables you usually want text to be left aligned and numbers to be right aligned?